Datadog.Serverless.Compat/DatadogServerlessCompat.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
```

Comment:
```suggestion
// <copyright file="DatadogServerlessCompat.cs" company="Datadog">
// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2025 Datadog, Inc.
// </copyright>

﻿using System;
```